src/controllers/UsersController.js

```diff
@@ -0,0 +1,74 @@
+import models from '../db/models';
+import TokenAuthenticate from '../helpers/tokenAuthentication';
+import Response from '../helpers/response';
+const { User, Profile } = models;
+import passwordHash from '../helpers/passwordHash';
+import EmailNotificationAPI from '../helpers/EmailNotificationAPI.js'; 
+
+
+/**
+ * @description Defines the actions to for the users endpoints
+ * @class UserController
+ */
+
+
+ class UserController {
```

Comment:
The name of the class is different from the name of the file, can you make them the same or do you have a reason for this?

Reply:
Ok, will do